### DISCUSSION TITLE
why the fuck do i have 2 notifications that don't exist

### DISCUSSION THREAD

**Discussion by StarmanAkremis** [community/community#58940] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

the title explains it, it says that i have 2 unread notifications but when i open the notifications there is nothing, this error already has months and is fucking annoying

**Comment by radar-blip** [comment]
Ghost notifications, see link below for fix:

https://github.com/orgs/community/discussions/6874#discussioncomment-3593547
